src/shapes/Polyline.ts

```diff
@@ -323,31 +323,21 @@ export class Polyline<
    * @static
    * @memberOf Polyline
    * @param {SVGElement} element Element to parser
-   * @param {Function} callback callback function invoked after parsing
    * @param {Object} [options] Options object
    */
-  static fromElement(
-    element: SVGElement,
-    callback: (poly: Polyline | null) => any,
-    options?: any
-  ) {
-    if (!element) {
-      return callback(null);
-    }
+  static async fromElement(element: SVGElement, options?: any) {
     const points = parsePointsAttribute(element.getAttribute('points')),
       // we omit left and top to instruct the constructor to position the object using the bbox
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       { left, top, ...parsedAttributes } = parseAttributes(
```

Comment:
same, spread out points so they are not passed to options

Reply:
points are not in parsedAttributes here.